### PR TITLE
Representations api

### PR DESCRIPTION
**Pull request by iamharish**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @iamharish has signed the CLA. Thanks for the pull request!

File: src/main/java/com/box/sdk/Representation.java [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+     * @return representation name
+     */
+    public String getRepresentation() {
+        return this.representation;
+    }
+
+    /**
+     * Get representation's set of static properties to distinguish between subtypes of a given representation,
+     * for example, different sizes of jpg's. Each representation has its own set of properties.
+
```
Comment (KarthikShanmugasundaram) [inline src/main/java/com/box/sdk/Representation.java]: extra space may be removed.

File: src/main/java/com/box/sdk/Representation.java [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+import java.net.URL;
+
+import com.eclipsesource.json.JsonObject;
+import com.eclipsesource.json.JsonValue;
+
+/**
+ * The class represents one instance of a file representation.
+ */
+public class Representation {
+    private String representation;
```
Comment (KarthikShanmugasundaram) [inline src/main/java/com/box/sdk/Representation.java]: a space may be added

File: src/main/java/com/box/sdk/Representation.java [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+    public Status getStatus() {
+        return this.status;
+    }
+
+    /**
+     * A set of static properties to distinguish between subtypes of a given representation,
+     * for example, different sizes of jpg's. Each representation has its own set of properties.
+     */
+    public class Properties {
+        private String dimensions;
```
Comment (KarthikShanmugasundaram) [inline src/main/java/com/box/sdk/Representation.java]: a space may be added

File: src/main/java/com/box/sdk/Representation.java [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+         */
+        public String getUrlTemplate() {
+            return this.urlTemplate;
+        }
+    }
+
+    /**
+     * Representation's status.
+     */
+    public class Status {
```
Comment (KarthikShanmugasundaram) [inline src/main/java/com/box/sdk/Representation.java]: Can this be a enum?

**Pull request merged**